src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -45,6 +45,16 @@
  */
 class FrameworkBundleAdminController extends Controller
 {
+    /**
+     * Used for success flash messages
+     */
+    protected const FLASH_TYPE_SUCCESS = 'success';
+
+    /**
+     * Used for error flash messages
+     */
+    protected const FLASH_TYPE_ERROR = 'error';
```

Comment:
```suggestion
    const FLASH_TYPE_ERROR = 'error';
```

Reply:
Why not protected?

Reply:
Because if you're in a module, you can't use this const :/
What if you retrieve the session, get all flash messages and want to parse only success message :)